.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -41,7 +41,9 @@ assignees: ''
 - [ ] create source tarballs (tar.gz and tar.xz) (@PaulWessel)
 - [ ] create macOS bundle (@PaulWessel)
 - [ ] create Windows installers (win32 and win64) (@joa-quim)
-- [ ] check if the source tarballs, macOS bundle and Windows installers work well
+- [ ] check if the source tarballs for Linux work well (@Esteban82, @anbj)
+- [ ] check if the macOS bundles work well (@seisman, @meghanrjones)
+- [ ] check if the Windows installers work well (@seisman)
```

Comment:
I don't have a Windows machine, so can't test the Windows installers.

Reply:
OK, wasn't sure.  Do we know anyone we could rope into helping with such a check?

Reply:
Changed it to (volunteers needed!)